webapp/pom.xml

```diff
@@ -94,7 +94,7 @@
          <plugin>
             <groupId>org.mortbay.jetty</groupId>
             <artifactId>maven-jetty-plugin</artifactId>
-            <version>6.1.10</version>
+            <version>6.1.26</version>
```

Comment:
I agree... with the change in master to servlet 3, we might as well go to jetty 8. But may be we need a separate ticket? I made the below change to the webapp/pom.xml and everything seems to work with master as expected. Not sure if it solves Ryan's classloader problem, which this ticket is about.
<groupId>org.mortbay.jetty</groupId>
<artifactId>jetty-maven-plugin</artifactId>
<version>8.1.16.v20140903</version>